.agents/session-2025-12-05-package-readme.md

```
@@ -0,0 +1,505 @@
+- Files: `PackageTests.Baseline_Qwiq.*.contents.verified.txt`
+
+**Example Changes**:
+
+**Manifest** (`PackageTests.Baseline_Qwiq.Core#manifest.verified.nuspec`):
+```xml
+<!-- ADDED -->
+<readme>README.md</readme>
+```
+
+**Contents** (`PackageTests.Baseline_Qwiq.Core#contents.verified.txt`):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)